### ISSUE TITLE
시스템14 성능 실험

### ISSUE THREAD

**Issue by februa22** [nlp-research/biaffineparser#55]

**Comment by februa22** [comment]
다음과 같이 워드 벡터 표현과 태그 벡터 표현을 concat. 해서 어절 벡터 표현을 생성:

- 워드 벡터 표현:
  - Pre-trained on large corpus
  - trainable=False
  - BiLSTM으로 200차원의 벡터 표현 생성
- 태그 벡터 표현:
  - Pre-trained on large corpus
  - trainable=False
  - BiLSTM으로 100차원의 벡터 표현 생성

Label을 디코딩할 때 CRFs를 사용.

MLP layer 1개 추가.

**Comment by februa22** [comment]
```json
{
    "num_lstm_units": 400,
    "num_lstm_layers": 3,
    "arc_mlp_units": 500,
    "label_mlp_units": 100,
    "word_embed_size": 200,
    "pos_embed_size": 100,
    "optimizer": "adam",
    "learning_rate": 0.001,
    "decay_factor": 0.9,
    "num_train_epochs": 100,
    "train_filename": "data/sejong.train.csv",
    "dev_filename": "data/sejong.test.csv",
    "out_dir": "parser/model_iss55",
    "word_vocab_name": "word.pkl",
    "pos_vocab_name": "pos.pkl",
    "rel_vocab_name": "rel.pkl",
    "head_vocab_name": "head.pkl",
    "word_embed_file": "embeddings/words.pos.original.vec",
    "pos_embed_file": "embeddings/words.tag.original.vec",
    "word_embed_matrix_file": "embeddings/word_embed_matrix.txt",
    "pos_embed_matrix_file": "embeddings/pos_embed_matrix.txt",
    "embed_dropout": 0.33,
    "lstm_dropout": 0.33,
    "mlp_dropout": 0.33,
    "batch_size": 128,
    "device": "gpu",
    "debug": false
}
```

**Comment by februa22** [comment]
성능:

> epoch: 7 - loss: 0.7946164608001709 - uas: 0.9065126180648804 - las: 0.8939075469970703 - eval_loss: 1.646679401397705 - eval_uas: 0.9061422944068909 - eval_las: 0.8812950849533081
